alexa.utterances

```diff
@@ -648,3 +648,25 @@ WhatNewShows what shows do you have
 WhatNewShows what shows i have
 WhatNewShows what shows we have
 WhatNewShows what shows you have
+WatchPVRBroadcast watch the program called {Broadcast}
+WatchPVRBroadcast play the program called {Broadcast}
+WatchPVRBroadcast find the broadcast called {Broadcast}
+WatchPVRBroadcast watch the broadcast called {Broadcast}
+WatchPVRBroadcast play the broadcast called {Broadcast}
+WatchPVRBroadcast look for the program called {Broadcast}
+WatchPVRBroadcast look for the broadcast called {Broadcast}
+WatchPVRBroadcast look for {Broadcast}
+WatchPVRBroadcast where is the program called {Broadcast}
+WatchPVRBroadcast where is the broadcast called {Broadcast}
+WatchPVRBroadcast where is {Broadcast}
+WatchPVRBroadcast find the program called {Broadcast}
+WatchPVRBroadcast find the broadcast called {Broadcast}
+WatchPVRBroadcast find {Broadcast}
```

Comment:
"look for {Broadcast}," "where is {Broadcast}," and "find {Broadcast}" shouldn't yet exist.

Currently, by default (without qualifiers on the media type), music artist is assumed when no type is specified.

This will change eventually, but for now, all other media types need to be specified in the command.